src/main/java/StringAddCal.java

```diff
@@ -0,0 +1,52 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCal {
+    public static int splitAndSum(String input) throws RuntimeException {
+        if (nullOrEmpty(input) == false) {
+         return 0;
+        }
+
+        String[] split = customDelimeter(input);
+
+        return calAllSum(split);
+    }
+
+    public static String[] customDelimeter(String input) {
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(input);
+        if(m.find()) {
```

Comment:
코드 포맷팅 확인해주시면 좋을 것 같습니다.
```suggestion
        if (m.find()) {
```